Changes

```diff
@@ -22,6 +22,9 @@ be mentioned in the 4.06 section below instead of here.)
 
 ### Tools:
 
+- MPR#7635: add an identifier to module and module type elements
```

Comment:
should mention "ocamldoc"

Reply:
Fixed, and added the missing reviewers field too.